src/documents/tests/test_workflows.py

```diff
@@ -2669,3 +2669,46 @@ def test_workflow_webhook_action_consumption(self, mock_post):
                 )
 
         mock_post.assert_called_once()
+
+    @mock.patch("httpx.post")
+    def test_send_webhook_data_and_json(self, mock_post):
```

Comment:
I'm assuming because this is old style, pytest-httpx didn't work as a fixture?

Reply:
Oh I didn’t try that, figured it was pretty simple to cover like this, happy to if you prefer. Assuming you’re good with the change itself

Reply:
Ok yea, apparently because this uses APITestCase the pytest_httpx fixtures dont work =/

Reply:
Answered my on question there, just move that one to its own class, its a bit different anyway so makes sense